src/stores/connectors/keys/delete.go

```diff
@@ -0,0 +1,26 @@
+package keys
+
+import (
+	"context"
+	"github.com/consensys/quorum-key-manager/src/stores/store/database"
+)
+
+func (c Connector) Delete(ctx context.Context, id string) error {
+	logger := c.logger.With("id", id)
+	logger.Debug("deleting key")
+
+	err := c.db.RunInTransaction(ctx, func(dbtx database.Database) error {
+		err := dbtx.Keys().Delete(ctx, id)
+		if err != nil {
+			return err
```

Comment:
This error will be handled to the users, we should be logging here and rephasing the message to something like "failed to persist key"